docs/troubleshooting.md

```diff
@@ -11,3 +11,24 @@ CMake Failures
 
   For development, if you simply want to disable HTTPS support entirely,
   pass the `-DUSE_HTTPS=OFF` argument to `cmake` when configuring it.
+
+* **`-- Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE)`**
+  Make sure that PkgConfig is installed on your system. It is a tool
+  used to retrieve information about installed libraries. If PkgConfig
+  is not installed, you can usually install it using your package manager.
+  (sudo apt-get install pkg-config).
+
+* **`-- Could NOT find GSSAPI (missing: GSSAPI_LIBRARIES GSSAPI_INCLUDE_DIR)`**
+  GSSAPI is the Generic Security Services API, and it seems the build system
+  cannot find the necessary libraries and include files. Install the GSSAPI
+  development package using your package manager. The package names may vary
+  depending on your operating system. For example, on Debian-based systems, you
+  can use: (sudo apt-get install libgssapi-dev) or (sudo apt install libgssglue-dev).
+
+* **'-- Could NOT find OpenSSL, try to set the path to OpenSSL root folder in the system
+  variable OPENSSL_ROOT_DIR (missing: OPENSSL_CRYPTO_LIBRARY OPENSSL_INCLUDE_DIR) logo`**
```

Comment:
Can we keep this to one line? Just the `Could NOT find OpenSSL` should be enough.

Reply:
Yes, Please